packages/wallets/censo/src/adapter.ts

```diff
@@ -124,15 +124,20 @@ export class CensoWalletAdapter extends BaseSignerWalletAdapter {
 
     async sendTransaction(
         transaction: Transaction,
-        connection: Connection,
+        _connectionContext: ConnectionContext,
         options: SendTransactionOptions = {}
     ): Promise<TransactionSignature> {
         try {
             const wallet = this._wallet;
             if (!wallet) throw new WalletNotConnectedError();
 
             try {
-                return await wallet.sendTransaction(transaction, connection, options);
+                return await wallet.sendTransaction(
+                    transaction,
+                    // @ts-ignore
+                    undefined, // This parameter is not used. See https://www.npmjs.com/package/@censo-custody/solana-wallet-adapter?activeTab=code
+                    options
+                );
```

Comment:
Are you OK with this, @bholzman, or does `window.censo` actually make use of this parameter? It doesn't look like `@censo-custody/solana-wallet-adapter` does.

Reply:
Yes. The censo solana wallet was eol’d, this entire adapter can be removed

Reply:
ty.